src/app/api/login/route.ts

```diff
@@ -37,8 +37,8 @@ export const POST = withAppRouterHighlight(async (request: NextRequest) => {
       .executeTakeFirst();
 
     if (!existingUser) {
-      return new Response("Wrong password or user does not exist!", {
-        status: 400,
+      return new Response("User does not exist!", {
```

Comment:
you should kept the old error message
`Wrong password or user does not exist!`, this is a standard practice in the industry. (https://www.gnucitizen.org/blog/username-enumeration-vulnerabilities/)